tests/components/autocomplete_dropdown.test.ts

```diff
@@ -39,7 +39,7 @@ beforeEach(async () => {
   ({ model, fixture } = await mountSpreadsheet());
 
   // start composition
-  await keyDown("Enter");
+  await keyDown(document.activeElement!, { key: "Enter" });
```

Comment:
@pro-odoo  let my know what you think. If we agree that the whole point here is to be specific about the targets of the tests, is it correct to assume that document.activeElement is indeed the right recipient of our event?